e2e/test/scenarios/filters/reproductions/31340-long-column-name-search-results.cy.spec.js

```diff
@@ -0,0 +1,86 @@
+import {
+  assertDescendantNotOverflowsContainer,
+  assertIsEllipsified,
+  getBrokenUpTextMatcher,
+  popover,
+  restore,
+} from "e2e/support/helpers";
+import { SAMPLE_DATABASE } from "e2e/support/cypress_sample_database";
+import { SAMPLE_DB_ID, SAMPLE_DB_SCHEMA_ID } from "e2e/support/cypress_data";
+
+const { PEOPLE_ID } = SAMPLE_DATABASE;
+
+const LONG_COLUMN_NAME =
+  "Some very very very very long column name that should have a line break";
+describe("issue 31340", function () {
```

Comment:
New line looks missing

```suggestion
  "Some very very very very long column name that should have a line break";

describe("issue 31340", function () {
```